doc/riverctl.1.scd

```diff
@@ -347,8 +347,17 @@ matches everything while _\*\*_ and the empty string are invalid.
 
 ## CONFIGURATION
 
-*attach-mode* *top*|*bottom*
-	Configure where new views should attach to the view stack.
+*default-attach-mode* *top*|*bottom*|*aside <N>*
```

Comment:
This rename is still a breaking change and needs to be reverted

Reply:
I am thinking about adding `attach-mode` as an alias to `default-attach-mode` to keep both to keep both consistency (with options like default-layout/output-layout) as well as compatibility with current configurations. What do you think?

Reply:
Sure, an alias would be fine.

Reply:
FYI, the change is up. You may review everything if you haven't already.